README.md

```diff
@@ -23,6 +23,19 @@ This dataset was prepared by Pierre-Luc Carrier and Aaron Courville, as part of
 
 Past Kaggle competition for this partical challenge: https://www.kaggle.com/c/challenges-in-representation-learning-facial-expression-recognition-challenge
 
+##To install
+--On Linux
```

Comment:
ja,

make sure cmake is up to scratch (older ubuntu/debian needs a cmake upgrade, v3.x instead of 2.x, may not be available in official repos), and install boost as system lib/package